src/Microsoft.DotNet.Cli.Utils/Muxer.cs

```diff
@@ -33,11 +33,24 @@ public Muxer()
             }
         }
 
+        public static string GetDataFromAppDomain(string propertyName)
+        {
+            var appDomainType = typeof(object).GetTypeInfo().Assembly?.GetType("System.AppDomain");
+            var currentDomain = appDomainType?.GetProperty("CurrentDomain")?.GetValue(null);
+            var deps = appDomainType?.GetMethod("GetData")?.Invoke(currentDomain, new[] { propertyName });
```

Comment:
I couldn't find a dependency that supports both net45 and netstandard1.5 that can do `AppContext.GetData`. `System.AppContext` supports net46.